src/client/components/ModalViewComponent/AdminPanel.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-export default function AdminPanel() {
+export default function AdminPanel({ owner }) {
   return (
     <div className="modal-content">
       <div className="modal-body">
         <img
           className="round"
-          src="https://randomuser.me/api/portraits/men/79.jpg"
-          alt="member"
+          src={require(`../../assets/images/demo-members-photos${owner.photoUrl}`)}
```

Comment:
I think the owner image will come from the members request also, right?

Reply:
Yes, I thought that way by maybe adding photoUrl in select query of that get all members endpoint.

Reply:
@senner007 Lint is not accepting use of require here. How can I modify it to get rid of it?

Reply:
I have no idea how I'm gonna solve this lint error. Googled it but they are not working.

Reply:
Should this url not come from the databse? The photo url?

Reply:
Yes

Reply:
without using require it is not working but when I add required lint errors come up

Reply:
Again, I don't see the need to use require here. Only owner.photoUrl

Reply:
When I use this `src={`../../assets/images/demo-members-photos${member.photoUrl}`}` it looks like : 
![image](https://user-images.githubusercontent.com/75583909/130466606-b77351b4-8280-4452-ba91-7b21cee14815.png)

Reply:
Even if I use a json file like this : 
```
[
  {
    "userId": 1,
    "role": "basic",
    "fullName": "Darrell Steward",
    "photoUrl": "../../assets/images/demo-members-photos/member1.jpg"
  },
  {
    "userId": 2,
    "role": "basic",
    "fullName": "Kristin Watson",
    "photoUrl": "../../assets/images/demo-members-photos/member2.png"
  },
  {
    "userId": 3,
    "role": "owner",
    "fullName": "Theresa Webb",
    "photoUrl": "../../assets/images/demo-members-photos/member3.png"
  },
  {
    "userId": 4,
    "role": "basic",
    "fullName": "Dianne Russell",
    "photoUrl": "../../assets/images/demo-members-photos/member4.png"
  }
]
```
and use this `<img className="round" src={member.photoUrl} alt="member" />` in Member component, it doesn't work...

Reply:
Can't you just hardcode to a real url in the json file for now?

Reply:
Like how? What do you mean by real url?

Reply:
online photos

Reply:
Then is this gonna be changed later related to database?

Reply:
@senner007 Can I have an answer here please?

Reply:
You will get the photo urls from the users from the database right. What you are using now is just a fake json to  display in storybook

Reply:
So if I use `src={owner.photoUrl}` only, are we gonna be able to retrieve images when the real data is sent from db?

Reply:
I won't actually, I wanted to adapt it related to real database but it won't work.

Reply:
When the use is created in our database we upload the photo url from the firebase data in the browser. Therefore each user will have a column with their url pointing to their photo

Reply:
Oh ok. Then should I leave like I did in last commit?

Reply:
No lets merge it. Then you can see how it works later